src/esn.calendar.libs/app/components/event-date-edition/event-date-edition.controller.js

```diff
@@ -113,13 +121,22 @@ function calEventDateEditionController(esnI18nDateFormatService, esnDatetimeServ
     _syncEventDateTime();
     _calcDateDiff();
     _onDateChange();
+    _refreshTimeInputs();
   }
 
   function onEndDateTimeChange() {
+    if (self.isMobile) {
+      self.end.set({
+        hour: self.endTime.getHours(),
+        minute: self.endTime.getMinutes()
+      });
+    }
```

Comment:
Why this logic?

Reply:
the `onEndDateTimeChange` handler can be called by both the time selector and the mobile picker.

if we are in the mobile picker we get the hours and minutes from the `Date` object ( not `moment` object ).

the native time picker uses only `Date` objects and due to the differences between moment and Date I have to only set the Hours and minutes without worrying about the Timezone.